thirdeye/thirdeye-pinot/pom.xml

```diff
@@ -17,6 +17,12 @@
   </properties>
 
   <dependencies>
+    <!-- https://mvnrepository.com/artifact/nz.ac.waikato.cms.weka/weka-dev -->
```

Comment:
Don't check in this library. Please also consult @kishoreg about open source licensing before using it.